src/blocks/VideoEmbedBlock/VideoEmbedBlock.astro

```diff
@@ -82,23 +97,56 @@ function getVideoUrl({ loop, mute, video }: { loop: boolean; mute: boolean; vide
 <figure>
   <video-embed
     data-autoplay={autoplay ? 'true' : undefined}
-    data-video-url={getVideoUrl({ loop: Boolean(loop), mute: Boolean(mute), video })}
+    data-provider={video.provider}
+    data-video-url={getVideoUrl({
+      loop: Boolean(loop),
+      mute: Boolean(mute),
+      video,
+    })}
     style={{ aspectRatio }}
   >
-    <a href={video.url} target="_blank" rel="noopener noreferrer">
+    <PlayButton href={video.url}>
       <img
-        alt={t('play_video_title', { title }) }
+        alt={t('play_video_title', { title })}
         loading="lazy"
         src={getImageUrl({ video })}
         srcset={getImageSrcSet({ video })}
         style={{ aspectRatio }}
       />
-    </a>
-    <iframe hidden allowfullscreen allow="autoplay" title={ title } style={{ aspectRatio }}></iframe>
+    </PlayButton>
+
+    <div class="consent-alert" role="alert" hidden>
+      <p class="consent-alert__text">
+        {t('consent_message_service', { service: providerName })}
```

Comment:
How do we handle content changes? Do we use migrations for content or just for content model changes?

Reply:
Is on the backlog, see #27. So for now, manually.